src/commands/flow/index.ts

```diff
@@ -12,9 +12,7 @@ export async function processFlow(command: Command, sasjs?: SASjs) {
   const csvFile = command.getFlagValue('csvFile') as string
   const targetName = command.getFlagValue('target') as string
   const { target } = await findTargetInConfiguration(targetName)
-  const logFolder =
-    (command.getFlagValue('logFolder') as string) ||
-    path.join(await getProjectRoot(), 'sasjsbuild', 'logs')
+  const logFolder = command.getFlagValue('logFolder') as string
```

Comment:
the idea was to provide a default log folder, if it wasn't provided

Reply:
It's being set in validateParams along with csvFile checks